File: 800/1919A/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int a = sc.nextInt();
            int b = sc.nextInt();
            
            if(((Math.abs(a-b)+1) & 1) == 0)
                System.out.println("Alice");
            else
                System.out.println("Bob");
        }
    }
}

File: 800/1919A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int a, b;
        cin >> a >> b;

        if (((abs(a - b) + 1) % 2) == 0)
            cout << "Alice\n";
        else
            cout << "Bob\n";
    }

    return 0;
}
